### DISCUSSION TITLE
Change density of home page

### DISCUSSION THREAD

**Discussion by woobenjamin** [mealie-recipes/mealie#983] [Feature Request]
Hi, I'm hoping that you could introduce the ability to show smaller, but more, recipes on a given page. For example, right now, it shows 3 recipe across, I'd be interested to seeing what it would look like if it were 5 or 6 across, so I can find a recipe faster by scrolling, rather than using the search.

Thanks. and Thank you for making Mealie!

**Comment by Nelinski** [comment]
I was wondering this too. Would be nice to get a slider at the top of the screen so you can choose between a list view, smaller images, medium images to the current large images.

**Comment by michael-genson** [comment]
Mealie has a condensed view now:
![image](https://github.com/user-attachments/assets/e3e53885-1117-4fc0-acc6-b269d2050c6a)

**Comment by woobenjamin** [comment]
Thank you!!!
